config/application.rb

```diff
@@ -47,5 +47,7 @@ class Application < Rails::Application
         resource '/timelines/*/manifest.json', headers: :any, methods: [:get, :post]
       end
     end
+
+    config.active_storage.service = (Settings&.active_storage&.service.presence || "local").to_sym
```

Comment:
I'm curious what does `.presence` do here?

Reply:
`presence` returns nil when it's an empty string, since `("" || "local" = "").to_sym = :""`